From dad2db1c6a5c596c8ec7ab4a1b4030643c294edc Mon Sep 17 00:00:00 2001
From: Jiri Konecny <DragonLichcz@gmail.com>
Date: Mon, 10 Oct 2016 10:49:38 +0200
Subject: [PATCH] Fix GUI layouts on Wayland

This is just a hotfix and it should be fixed properly in the code later.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2242fa6..4f2d134 100644
--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,7 @@ build:
 	$(CC) ./src/check_direct_rendering.c -o ./bin/playonlinux-check_dd
 	$(PYTHON) ./python/*.py
 	$(PYTHON) ./python/lib/*.py
-	echo -e '#!/bin/bash\n${sharedir}/playonlinux/playonlinux "$$@"\nexit 0' > ./bin/playonlinux
+	echo -e '#!/bin/bash\nGDK_BACKEND=x11 ${sharedir}/playonlinux/playonlinux "$$@"\nexit 0' > ./bin/playonlinux
 	echo -e '#!/bin/bash\n${sharedir}/playonlinux/playonlinux-pkg "$$@"\nexit 0' > ./bin/playonlinux-pkg
 	chmod +x ./bin/playonlinux
 	chmod +x ./bin/playonlinux-pkg
-- 
2.7.4

